### PR TITLE
[US-627] Example for Api Key Usage logs

### PR DESCRIPTION
**Pull request by kellemNegasi**

### CODE REVIEW

**Review by anovik — APPROVED** [review]
LGTM

File: license/unipdf_license_key_usage_log.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+	license.SetMeteredKeyUsageLogVerboseMode(true)
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelInfo))
+	license.SetMeteredKeyPersistentCache(false)
```
Comment (sampila) [inline license/unipdf_license_key_usage_log.go:31]: Probably can remove this? as this only needed when to report the usage to unicloud server immediately

**Review by sampila — CHANGES_REQUESTED** [review]
added a comment

File: license/unipdf_license_key_usage_log.go (hunk 1 above)
Comment (kellemNegasi) [reply license/unipdf_license_key_usage_log.go:31]: Do you mean this one 
`license.SetMeteredKeyPersistentCache(false)` ?

Comment (sampila) [reply license/unipdf_license_key_usage_log.go:31]: Yes

**Review by sampila — APPROVED** [review]
Looks good to me

**Pull request merged**